website/docs/topics/non-openai-models/cloud-based-proxy-servers.md

```diff
@@ -0,0 +1,187 @@
+# Cloud-based Proxy Servers
```

Comment:
Same comment as in the local proxy server: we can rename this page "Together AI", and move the first two paragraphs here to the `about-using-non-openai-models.md`.